build_product_images.py

```diff
@@ -1,35 +1,47 @@
 #!/usr/bin/env python
 """
-Build and possibly publish product images. It doesn't login to any registry when publishing
-but it assumes a `docker login` has been performed before.
+Build and possibly publish product images.
```

Comment:
Did a quick `pylint build_product_images.py`. You can work on those if you want to but you don't have to.
```
build_product_images.py:77:15: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
build_product_images.py:104:46: E1136: Value 'list' is unsubscriptable (unsubscriptable-object)
build_product_images.py:104:51: E1136: Value 'list' is unsubscriptable (unsubscriptable-object)
build_product_images.py:148:12: W1510: Using subprocess.run without explicitly set `check` is not recommended. (subprocess-run-check)
build_product_images.py:154:4: W0621: Redefining name 'product_to_build' from outer scope (line 153) (redefined-outer-name)
build_product_images.py:170:4: R1705: Unnecessary "else" after "return" (no-else-return)
build_product_images.py:28:0: C0411: standard import "import argparse" should be placed before "import conf" (wrong-import-order)
build_product_images.py:29:0: C0411: standard import "import subprocess" should be placed before "import conf" (wrong-import-order)
```

Reply:
Resolved some of them, thanks for your hint.